BraveRewardsUI/Tipping/TippingSelectionViewController.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import BraveRewards
+
+class TippingSelectionViewController: OptionsSelectionViewController {
+  
+  private let state: RewardsState?
```

Comment:
Once changes are made to `OptionsSelectionViewController` as explained above, you won't need a private init and can make this non-optional, since its assigned in init

Reply:
Fixed.